homeassistant/components/aprs/device_tracker.py

```diff
@@ -57,7 +57,10 @@
 
 def make_filter(callsigns: list) -> str:
     """Make a server-side filter from a list of callsigns."""
-    return " ".join("b/{0}".format(cs.upper()) for cs in callsigns)
+    call_filter = ""
+    for cs in callsigns:
+        call_filter += f" b/{cs.upper()}"
+    return call_filter
```

Comment:
Not sure if there is a better way to do this with f-strings, could not find an example online either.
Open for suggestions :)

Reply:
Solved in https://github.com/home-assistant/home-assistant/pull/32110/commits/777d764f137e2ef6fc7ddfbe63571ceceb9ae5c1